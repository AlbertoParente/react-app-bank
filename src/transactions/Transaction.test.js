import React from 'react';
import { render } from '@testing-library/react';
import Transaction from './Transaction';

describe('Extract transaction component', () => {
    it('Should always keep the same snapshot', () => {
        const { container } = render(
            <Transaction
                date="02/04/2022"
                type="withdraw"
                value="20.00"
            />)

        expect(container.firstChild).toMatchSnapshot();
    });
});
