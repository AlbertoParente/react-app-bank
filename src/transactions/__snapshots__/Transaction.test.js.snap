// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Extract transaction component Should always keep the same snapshot 1`] = `
<div
  class="transaction-container"
>
  <p>
    02/04/2022
  </p>
  <p>
    withdraw
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
